src/config/networks/__tests__/networks.test.ts

```diff
@@ -0,0 +1,109 @@
+import fs from 'fs'
+
+import { NetworkConfig } from 'src/config/networks/network.d'
+import { ETHEREUM_NETWORK } from 'src/logic/wallets/getWeb3'
+import { isValidURL } from 'src/utils/url'
+
+describe('Network config file test', () => {
```

Comment:
shouldn't it be Network**s** config file**s**?

Reply:
yes, makes sense